vast-perf.sh

```diff
@@ -434,15 +446,31 @@ if  [ -f "/vast/vman/mgmt-vip" ] && [ $NOT_CNODE == 0 ]; then
   IS_VAST=1
   mVIP=`cat /vast/vman/mgmt-vip`
   echo "running on a Vast node. setting VMSIP to ${mVIP}"
-  #next block is commented out. as of newer vast-os, rdma should work.
-  # if [ ${PROTO} == "rdma" ] && [ ${FORCE_RDMA} == "0" ]; then
-  #   echo "on a cnode, not using rdma, falling back to tcp."
-  #   PROTO=tcp
-  # fi
+
   if [ ${PROTO} == "multipath" ]; then
     echo "can't use multipath on cnode, falling back to regular rdma"
     PROTO=rdma
   fi
+
+  # Nowadays, vast-OS supports RDMA so default to it when running on loopback
+  # mode, but first check two exceptions: Broadcom NICs & Rocky
+
+  # Check for the Broadcom driver
+  LSMOD=`sudo lsmod|grep bnxt_en|wc -l`
+  if [ "$LSMOD" -gt "0" ]; then
+    PROTO=tcp
+    echo "setting PROTO to ${PROTO} because broadcom"
+
+  # Check for Rocky
+  elif grep -q "Rocky" /etc/redhat-release; then
+    PROTO=tcp
+    echo "setting PROTO to ${PROTO} because rocky"
+
+  # by default, use RDMA
+  else
+    PROTO=rdma
```

Comment:
it's probably no longer really relevant nowadays, so might not even be worth changing. thus, i'm saying this more for the sake of completeness: the actual reason for why rdma is working or not is likely not rocky linux vs not rocky linux, but rather whether an actually functioning rpcrdma kernel module is present or not:
```
selab-avnet-203 vastdata@selab-cb10-c1 ~:$ modinfo rpcrdma | grep dummy
description:    rpcrdma dummy kernel module
```

Reply:
Right, and my objective wasn't to change how that detection worked yet. I plan on doing that after they fix the RDMA client support in vastos (see SFDC case 00036159 -- no Jira ticket for it yet).